### PR TITLE
fixing the UI news showcasing

### PR DESCRIPTION
**Pull request by houssem-yahiaoui**
Fixing the Dyalna news showcasing and proposing a solution for the issue #11

### CODE REVIEW

File: src/controllers/dyalna.js [hunk 1]
```diff
@@ -5,14 +5,29 @@ const news = (bot, msg) => {
...
+              'color': '#36a64f',
+              'author_name': 'Dyalna Top News',
+              'author_link': 'dyalna.com',
+              'title': `${ post.title }`,
+              'title_link': `https://dyalna.com/post/show/${ post.slug }`,
+              'thumb_url': `${ post.picture }`,
+              'footer': 'Dyalna.com',
+              'footer_icon': 'https://www.dyalna.com/favicon.ico?2'
+            }],
+            'unfurl_links': false
```
Comment (youknowriad) [inline src/controllers/dyalna.js]: I guess this attribute is not useful anymore

File: src/controllers/dyalna.js [hunk 2]
```diff
@@ -5,14 +5,29 @@ const news = (bot, msg) => {
...
     .get('https://dyalna.com/api/post/top')
     .then(data => {
-      bot.startConversation(msg, (err,convo) => {
+      bot.startConversation(msg, (err, convo) => {
         convo.say('Sraw bezzaf swala7, voici les top news :')
         data.body.forEach(post => {
-          convo.say({ text: ` - ${ post.title } https://dyalna.com/post/show/${ post.slug }`, unfurl_links: false })
+          convo.say({
+            'attachments' : [{
+              'fallback': 'Required placeholder for news summary.',
```
Comment (youknowriad) [inline src/controllers/dyalna.js]: Should we use post.title for the fallback or something?

Comment (youknowriad) [reply src/controllers/dyalna.js]: Or maybe we could drop this attribute if not useful

File: .env.sample [hunk 3]
```diff
@@ -1,3 +0,0 @@
-# Generated at https://my.slack.com/services/new/bot
-# API_TOKEN =
-# API_TOKEN_DEV =
```
Comment (youknowriad) [inline .env.sample:3]: We should not delete this file :)

**Review by youknowriad — CHANGES_REQUESTED** [review]
Some small notes ;)

File: src/controllers/dyalna.js (hunk 2 above)
Comment (houssem-yahiaoui) [reply src/controllers/dyalna.js]: as it seems over the docs, its in a way or another a required field but apparently no harm on deleting it ! so yeah let's drop it !

**Review by IOAyman — DISMISSED** [review]
In addition to the comments by @youknowriad, it would be nice if `npm run lint` had no output 😉

**Comment by houssem-yahiaoui** [comment]
Yeah @IOAyman it was the case ! just three warnings coming missing the 3 warnings regarding missing a commas !

**Comment by houssem-yahiaoui** [comment]
@IOAyman @youknowriad i guess it's all good now.

**Review by youknowriad — APPROVED** [review]

**Pull request merged**